Layout inheritance exercise (#36)
carpentries-incubator/jekyll-pages-novice: _episodes/07-templates.md

- "You can avoid duplicated effort by basing new layouts on previous ones"
---

> ## Expanding the layout
>
> We will probably want to include the contact line we added in the previous section
> in every standard page on our site.

> > It would be better placed into the layout template,

Reviewer: ```suggestion
> Instead, we can place it in the layout template
```
Slight language change suggestion

> so we don't have to remember to `include` it every time we make a new page.
> Create a new layout, `page-contact.html`,
> which is based on the existing `page` layout and
> displays the contact information from `_includes/contact.md`
> after the page content.
> Check that this works by applying that layout to the `team.md` page.
>
> > ## Solution
> > Your new layout file, `_layouts/page-contact.html`,
> > should look like this:
> >
> > ~~~
> > ---
> > layout: page

> > > ---

Reviewer: ```suggestion
> > ---
> > 
```
Sometimes issues with rendering when there is no blank line after the header.

Author: well spotted, thanks!
